src/AddTodoForm.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        event.target.title.value = "";
+        props.onAddTodo(todoTitle);
+    }
+
+    return (
+        <form onSubmit={handleAddTodo}>
+            <label htmlFor="todoTitle">Title</label>
+            <input name="title" id="todoTitle" type="text" />
+            <button>Add</button>
```

Comment:
it is important to add a type to your button element, it helps with accessibility also you might have a form with multiple buttons to do other things in the form apart from just submitting the form, if you do not put a type to the button, all the buttons become a `submit` button and it is a common source of bugs.

so use the type `submit` to indicate that clicking on the button submits your form

```suggestion
            <button type='submit'>Add</button>
```